Modules/Microsoft365DSC/DSCResources/MSFT_AADRoleSetting/settings.json

```diff
@@ -0,0 +1,30 @@
+﻿{
```

Comment:
We updated this settings.json a while back. The Permissions section has been extended. Can you please check a different resource and make sure the same structure is used?

For example: https://github.com/microsoft/Microsoft365DSC/blob/Dev/Modules/Microsoft365DSC/DSCResources/MSFT_AADApplication/settings.json